packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/options/options.html

```diff
@@ -63,12 +63,14 @@
 
         <oui-tile-definition
             class="pb-2"
+            data-ng-if="$ctrl.bindings.options.basic.description.available"
```

Comment:
```suggestion
            data-ng-if="$ctrl.options.basic.description.available"
```
`bindings` is redundant

Reply:
This refers to https://github.com/ovh/manager/pull/3915/files#diff-951773e2879a3cda423ab6541357d04a386e81f2a3b675f7ed430c6ed6ad81d8R60
It should be `$ctrl.bindings.options.basic.description.available`